### DISCUSSION TITLE
Ruler to missing key on the header or on the body

### DISCUSSION THREAD

**Discussion by hitokiri** [mockoon/mockoon#496] [Help] [answered]
when we need to review if the header has a specific key the app not have the capacity to recognize those types of things

**Comment by 255kb** [comment, marked as answer]
Could you please elaborate a little bit, by providing an example of what you tried maybe? 
Mockoon has a rule system when with which you can serve different responses based on headers:
![image](https://user-images.githubusercontent.com/7489814/120660449-a272e580-c487-11eb-9ff2-1f0b0fb4be15.png)

**Reply by hitokiri** [reply]
yes, I splain more, the point here is, add a type of  Ruler to can manage missing keys on the Header case, because for now the ruler only manage if the key no have a value, but not work if this not exists; and can implement to the same way for the body JSON

**Reply by 255kb** [reply]
Ok I see, there is an open issue for this #255 . This is probably something we will work on in the coming months.
